source/codegen/metadata/nixnet/functions_addon.py

```diff
@@ -0,0 +1,2 @@
+functions_override_metadata = {
```

Comment:
Add a header comment similar to what you have for `enums_addon.py`. You can copy text from some of the other drivers that have it.